### PR TITLE
OSDOCS-4652: adding Additional information assembly and mods to Configuration

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
[OSDOCS-4652](https://issues.redhat.com//browse/OSDOCS-4652) adding Additional information assembly and mods to Configuration

Applies to: 4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4652

Link to docs preview:
https://53516--docspreview.netlify.app/microshift/latest/microshift_configuring/microshift-things-to-know.html

QE:
N/A- MicroShift is Developer Preview
SME lgtm was applied below

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@dhellmann PTAL. And please add anyone else who should review. Thank you!

**Comment by dhellmann** [comment]
@ShaunaDiaz We want to document that MicroShift restarts when the host IP changes or when the clock jumps, but I don't think we want to share the developer instructions for testing that behavior. There are a lot of details here about VMs that is only relevant for MicroShift developers.

**Comment by ShaunaDiaz** [comment]
@dhellmann Revised per our conversation, PTAL and let me know if /lgtm?

**Review by dhellmann — COMMENTED** [review]
/lgtm

I'm happy with this version. I have one nit, listed inline. We can deal with that in a follow-up or you could fix it if other reviewers have changes they'd like for you to make.

File: microshift_configuring/microshift-things-to-know.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="microshift-things-to-know"]
+= About responsive restarts and security certificates
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-configuring
+toc::[]
+
+{product-title} responds to system configuration changes and restarts after alterations are detected, including IP address changes, clock adjustments, and security certificate age.
```
Comment (snarayan-redhat) [inline microshift_configuring/microshift-things-to-know.adoc:8]:
```suggestion
{product-title} responds to system configuration changes, and restarts after alterations are detected, including IP address changes, clock adjustments, and security certificate age.
```

File: microshift_configuring/microshift-things-to-know.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+[id="microshift-ip-address-clock-changes_{context}"]
+== IP address changes or clock adjustments
+{product-title} depends on device IP addresses and system-wide clock settings to remain consistent during its runtime. However, these settings may occasionally change on edge devices, such as DHCP or Network Time Protocol (NTP) updates.
+
+When such changes occur, some {product-title} components may stop functioning properly. To mitigate this situation, {product-title} monitors the IP address and system time and restarts if either setting change is detected.
```
Comment (snarayan-redhat) [inline microshift_configuring/microshift-things-to-know.adoc:14]:
```suggestion
When such changes occur, some {product-title} components may stop functioning properly. To mitigate this situation, {product-title} monitors the IP address and system time, and restarts if either setting change is detected.
```

File: microshift_configuring/microshift-things-to-know.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+The threshold for clock changes is if the time is adjusted by greater than 10 seconds in either direction. Smaller drifts on regular time adjustments performed by the Network Time Protocol (NTP) service do not cause a restart.
```
Comment (snarayan-redhat) [inline microshift_configuring/microshift-things-to-know.adoc:16]: I think here`is` is not required
`{product-title} responds to system configuration changes and restarts after alterations are detected, including IP address changes, clock adjustments, and security certificate age.`

Comment (snarayan-redhat) [inline microshift_configuring/microshift-things-to-know.adoc:16]:
```suggestion
The threshold for clock changes is if the time is adjusted by greater than 10 seconds in either direction. Smaller drifts on regular time adjustments performed by the NTP service do not cause a restart.
```

File: modules/microshift-certificate-lifetime.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+. Short-lived certificates having certificate validity of one year.
+. Long-lived certificates having certificate validity of 10 years.
+
+Most server or leaf certificates are short-lived.
+
+An example of a long-lived certificate is the client certificate for `system:admin user` authentication, or the certificate of the signer of the `kube-apiserver` external serving certificate.
+
+[id="microshift-certificate-rotation_{context}"]
+== Certificate rotation
+As certificates age, {product-title} can be restarted to rotate certificates. A certificate that is close to expiring may also automatically cause a restart. Read the following situation overviews to understand the actions at each moment in time:
```
Comment (snarayan-redhat) [inline modules/microshift-certificate-lifetime.adoc:19]: Use `might instead of `may`

File: modules/microshift-certificate-lifetime.adoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+.. When a short-term certificate is 8 months old, it is rotated when {product-title} starts or restarts.
+.. When a long-term certificate is 9 years old, it is rotated when {product-title} starts or restarts.
+
+. Red zone
+.. When a short-term certificate is 8 months old, {product-title} restarts to rotate and apply a new certificate.
+.. When a long-term certificate is 9 years old, {product-title} restarts to rotate and apply a new certificate.
+
+[NOTE]
+====
+If the rotated certificate is a Certificate Authority, all of the certificates it signed rotate.
```
Comment (snarayan-redhat) [inline modules/microshift-certificate-lifetime.adoc:35]:
```suggestion
If the rotated certificate is a Certificate Authority, the certificates that it signed rotate.
```

File: modules/microshift-certificate-lifetime.adoc [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+====
+
+.Stop-light timeline of {product-title} certificate validity.
+image::microshift-cert-rotation.png[<{product-title} graph with symbolic green-yellow-red stoplight map of certificates>]
```
Comment (snarayan-redhat) [inline modules/microshift-certificate-lifetime.adoc:39]: The section name says `Stop-light`. Can you check if it has to be stop-light or stoplight?

**Review by snarayan-redhat — COMMENTED** [review]
Awesome job! Few comments and suggestions.

File: modules/microshift-certificate-lifetime.adoc (hunk 6 above)
Comment (ShaunaDiaz) [reply modules/microshift-certificate-lifetime.adoc:39]: IMB does not use the hypen. Good catch, thanks1

**Pull request merged**